Docs: Agregar resumen del capitulo 0

IIKUYY opened the pull request:
## what

- Agregar el resumen del capitulo 0

## why

- Continuar con las tareas dadas

## references
- http://diveintohtml5.info/index.html

IIKUYY committed:
Docs: Agregar resumen del capitulo 0

## what

- Agregar el resumen del capitulo 0

## why

- Continuar con las tareas dadas

## references
- http://diveintohtml5.info/index.html

semanticmx commented on Ch1/Ch1.md:
```diff
@@ -0,0 +1,26 @@
+# Introducción: Cinco cosas que tienes que saber sobre HTML5
```
para el primer archivo de cada folder podemos iniciar directamente con README.md para que github lo muestre por default.

semanticmx commented on Ch1/Ch1.md:
```diff
@@ -0,0 +1,26 @@
+
+Puedes llegar a preguntarte como empezar a usar HTML5 si los navegadores antiguos no tienen soporte para este, pero la cuestión es que HTML5 no es una cosa, es un conjunto de funciones, y puedes detectar si tiene soporte para alguna de estas como: `canvas`, `video` o `geolocation`.
+
+Talvez puedas pensar en HTML5 como `tags` y `angle brackets`, y son una parte importante, pero te estas perdiendo del todo, las especificaciones de HTML5 también defienen como esos `angle brackets` interactúan con JavaScript, atravez del `DOM` por sus siglas en ingles (Document Object Model). HTML5 no define los `tags` cada uno tiene un `DOM API`, puedes usar estas `API` para detectar que tipo de ...
```
acá faltó un espacio "Tal vez"

semanticmx replied:
"a través"

semanticmx commented on Ch1/Ch1.md:
```diff
@@ -0,0 +1,26 @@
+
+## No necesitas tirar nada
+
+HTML4 es ek sistema `markups` más exitosos, HTML5 se construyo en base a su exito, no tienes que eliminar los `markups` que tengas, ni tienes que volver a aprender cosas que ya sabes, si tienes alguna pagina en HTML4, va a funcionar en HTML5.
```
"ek"

semanticmx replied:
"éxito"

semanticmx commented on Ch1/Ch1.md:
```diff
@@ -0,0 +1,26 @@
+## No necesitas tirar nada
+
+HTML4 es ek sistema `markups` más exitosos, HTML5 se construyo en base a su exito, no tienes que eliminar los `markups` que tengas, ni tienes que volver a aprender cosas que ya sabes, si tienes alguna pagina en HTML4, va a funcionar en HTML5.
+Lo unico que trae HTML5 son mejoras, por ejemplo HTML4 tiene soporte para algunas opciones de controles, mientras qye HTML5 agrego algunas más opciones, como `sliders` y `date pickers`, esto sirve para actualizar tus paginas sin tener que rehacerlas.
```
qye

semanticmx commented on Ch1/Ch1.md:
```diff
@@ -0,0 +1,26 @@
+HTML4 es ek sistema `markups` más exitosos, HTML5 se construyo en base a su exito, no tienes que eliminar los `markups` que tengas, ni tienes que volver a aprender cosas que ya sabes, si tienes alguna pagina en HTML4, va a funcionar en HTML5.
+Lo unico que trae HTML5 son mejoras, por ejemplo HTML4 tiene soporte para algunas opciones de controles, mientras qye HTML5 agrego algunas más opciones, como `sliders` y `date pickers`, esto sirve para actualizar tus paginas sin tener que rehacerlas.
+
+## Facil de empezar
```
Fácil

semanticmx reviewed (changes requested):
muy bien, solo hay que corregir la parte de ortografía.

Esta es tan importante como el código, normalmente los clientes no revisan el código que genera las páginas, sino el contenido y cuando detectan algún error de ortografía impacta directamente a su marca ante sus clientes, entonces es algo que debemos cuidar mucho.

Yo recomiendo utilizar herramientas como los correctores ortográficos o grammarly para textos en inglés.

IIKUYY committed:
Chores: Corregir errores gramaticales

## what

- Realizar la corrección de errores gramaticales en el resumen

## why

- Mantener un estándar de calidad en el proyecto

## references
-

IIKUYY committed:
Feat: Agregar documento README.md en el capitulo 1

## what

- Agregar archivo README.md al capitulo 1

## why

- Mantener un formato en todos los capitulos

## references
-

IIKUYY commented:
Se realizaron los cambios requeridos
